src/tools/dev/xml/GenerateCMake.h

```diff
@@ -596,24 +591,9 @@ class CMakeGeneratorPlugin : public Plugin
         {
             CMakeWrite_TargetLinkDirs(out, "", "E", "_ser", eldflagsSer);
         }
-        if (!vtk8_elibsSer.empty())
+        if (!vtk9_elibsSer.empty())
```

Comment:
Is the `9` designation still needed on this variable?

Reply:
Probably not. This will all be rewritten with my CMake modernization efforts. Can it wait until then?

Reply:
Thats fine.